pyiron_atomistics/atomistics/structure/atoms.py

```diff
@@ -2566,17 +2566,63 @@ def set_initial_magnetic_moments(self, magmoms=None):
         Set array of initial magnetic moments.
 
         Args:
-            magmoms (numpy.ndarray/list): List of magneric moments
+            magmoms (numpy.ndarray/list/dict/float/None): List, dict or single
+                value of magneric moments
+
+
+        Example I:
```

Comment:
If it takes a an array, list, dict, float, it would probably be good to explicitly show what the behaviours are in each of the cases, but I'm just mostly nitpicking. I'll write up the other examples after shower/dinner